ui/perfherder/helpers.js

```diff
@@ -673,16 +664,20 @@ export const getTitle = alertSummary => {
   return title;
 };
 
-export const assignBug = (alertSummary, taskNumber, issueTrackerId) =>
-  update(getApiUrl(`/performance/alertsummary/${alertSummary.id}/`), {
+export const assignBug = async (alertSummary, taskNumber, issueTrackerId) => {
+  await update(getApiUrl(`/performance/alertsummary/${alertSummary.id}/`), {
     bug_number: taskNumber,
     issue_tracker: issueTrackerId,
-  }).then(() => refreshAlertSummary(alertSummary));
+  });
+  await refreshAlertSummary(alertSummary);
+};
 
-export const unassignBug = alertSummary =>
-  update(getApiUrl(`/performance/alertsummary/${alertSummary.id}/`), {
+export const unassignBug = async alertSummary => {
+  await update(getApiUrl(`/performance/alertsummary/${alertSummary.id}/`), {
     bug_number: null,
-  }).then(() => refreshAlertSummary(alertSummary));
+  });
+  await refreshAlertSummary(alertSummary);
```

Comment:
Does this `await` need to be here? Presumably, it just needs to wait for line 676's promise to return before executing this function?